src/ticketmap/index.tsx

```diff
@@ -444,7 +444,10 @@ export default class TicketMap extends Component<Props, State> {
 
   getAllPaths = (id) =>
     Array.from(this.mapRootRef.querySelectorAll(`[data-section-id${id ? `="${id}"` : ''}]`))
```

Comment:
```suggestion
    Array.from(this.mapRootRef.querySelectorAll(`path[data-section-id${id ? `="${id}"` : ''}]`))
```

Reply:
We don't want to do this because it would exclude groups without paths, or elements other than paths

Reply:
Isn't that what this PR is doing?

Reply:
The goal of this PR is to exclude nodes _when they have children_. If a node doesn't have children, we include it, else we exclude it and include it's children.

Reply:
For example a `rect` element doesn't have children, if it has a `data-section-id` attribute then we want to include it. A `group` element does have children, we would want to include its children and exclude the `group` element.

Reply:
👌